ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -60,22 +60,21 @@ export const defaultCromwellConfig: CreateAppRequest = {
   },
 };
 
-// visible/actionable logic is copied from runtime-utils
-
 export const isVisible = (status: AppStatus): boolean =>
-  status && ![AppStatus.DELETED, AppStatus.ERROR].includes(status);
+  status && ![AppStatus.DELETED].includes(status);
```

Comment:
This change makes ERROR state apps automatically visible, instead of requiring a click